doc/nrf/gs_updating.rst

```diff
@@ -14,6 +14,20 @@ You might also want to switch to a newer release or check out the latest state o
 However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
 For an overview of changes in the latest releases, see :ref:`release_notes`.
 
+.. _gs_updating_in_vsc:
+
+Updating |VSC|
+*****************
+
+Visual Studio Code checks for extension updates and automatically installs them when they are available.
+After an extension is updated, VS Code prompts you to reload the application.
+If you disabled automatic updates, open the :guilabel:`Extensions` tab and locate the |VSC| extension.
+The :guilabel:`Update` button appears when an update is available.
+Click on the button to install the update.
+
+The |VSC| extension lets you update west and the associated |NCS| repositories within its :guilabel:`Source Control` panel.
+For detailed instructions, see the `nRF Connect for Visual Studio Code`_ documentation site.
```

Comment:
Can you link to the related page or at least mention the name of the page?
```suggestion
For detailed instructions, see the page about updating the extension on the `nRF Connect for Visual Studio Code`_ documentation site.
```
(just an example)

Reply:
I have been holding off on this since we have been frequently changing the structure of the documentation as it grows. I can add the page in, but this may break often over the next few months as the site grows since we are on a more frequent release schedule than NCS.

Reply:
Updated with current link. We'll triage later if it breaks.